docs/tempo/website/guides/multitenancy.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Multitenancy
+---
+
+Tempo is a multitenant distributed tracing backend. It supports multitenancy through the use
+of a header: `X-Scope-OrgID`. This guide details how to setup or disable multitenancy.
+
+## Multitenancy
+
+If you're interested in setting up multitenancy, please consult the [multitenant example](https://github.com/grafana/tempo/tree/master/example/docker-compose/docker-compose.multitenant.yaml)
+in the repo. This example uses the following settings to achieve multitenancy in Tempo:
+
+- Configure the OTEL Collector to attach the X-Scope-OrgID header on push:
```

Comment:
Should these be numbered steps, not bullets? Or can a user configure these settings in any order? Thanks.

Reply:
There is no implied order here.  Bullets are fine.